scalableminds/webknossos
# Grouped Dataset Actions into a Dropdown

By suggestion from Johannes, I grouped the less frequently used dataset actions into an overflow menu / dropdown to make more room for other buttons etc.

![image](https://user-images.githubusercontent.com/1105056/29224613-5ace9cb6-7ecb-11e7-92dd-ded8962c41bd.png)

### Mailable description of changes:
- Grouped less frequently used dataset actions into an overflow menu / dropdown to make more room for other buttons etc

### Steps to test:
- Start tracing
- Check that all dropdown actions / functionalities still work
- Check layouting for tasks ("Finish and Get Next Task" Button)
- Check layouting for read only tracings

------
